src/components/button/button.yaml

```diff
@@ -0,0 +1,14 @@
+variants:
+- name: default
+  data:
+    text: Save and continue
+    classes: 'boom'
+- name: disabled
+  data:
+    text: Disabled button
+    isDisabled: true
+- name: start
+  data:
+    text: Start now
+    url: /
+    isStart: true
```

Comment:
Is `is` needed in `isStart` and `isDisabled`?
On GOV.UK we tried to mirror the DOM where possible, "disabled" would follow that approach.

Reply:
I can do either way with this one. Seeing we're proposing is- prefix for component states I'd like to follow similar convention. We will be testing this on devs.